core-services/prow/02_config/_config.yaml

```diff
@@ -2231,6 +2228,208 @@ tide:
     - do-not-merge/invalid-owners-file
     repos:
     - opendatahub-io/odh-manifests
+  - includedBranches:
+    - release-4.5
```

Comment:
This should include also the `openshift-4.5` branch (which also needs to be removed from the queries above that include it)